### ISSUE TITLE
ProjectRed Transportation

### ISSUE THREAD

**Issue by LeviSimioni** [GTNewHorizons/GT-New-Horizons-Modpack#14853]
### Your GTNH Discord Username

_null__

### Your Pack Version

2.4.0

### Your Server

SP

### Java Version

Java 8

### Type of Server

Single Player

### Your Expectation

i WANT to connect extraction pipe to a gregtech machine 

### The Reality

https://youtu.be/lytFlkHXDM0 here's my problem with this pipes 

### Your Proposal

fix it

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the bug still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I can reproduce this problem consistently by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.
- [X] I have asked other people and they confirm they also have this problem by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.

**Comment by lordIcocain** [comment]
Did you try rotate output side to pipe? also why not using auto output?

**Comment by LeviSimioni** [comment]
yes, I tried, I put it in auto output mode, but it still didn't work
